src/main/java/de/tum/in/www1/artemis/domain/enumeration/ExerciseLifecycle.java

```diff
@@ -13,27 +15,47 @@ public enum ExerciseLifecycle implements IExerciseLifecycle {
         public ZonedDateTime getDateFromExercise(Exercise exercise) {
             return exercise.getReleaseDate();
         }
+
+        @Override
+        public ZonedDateTime getDateFromQuizBatch(QuizBatch quizBatch, QuizExercise quizExercise) {
+            return getDateFromExercise(quizExercise);
+        }
```

Comment:
just double-checking: is this Method correct? We want to get the release date for the quiz batch and call `getDateFromExercise`.